packages/cln_plugin/test/cln_plugin_method_test.dart

```diff
@@ -96,5 +117,9 @@ void main() {
       expect(getManifest.containsKey('hooks'), true);
       expect(getManifest.containsKey('subscriptions'), true);
     });
+
+    group('Init test: ', () {
+      test('Property test - dynamic', () {});
+    });
   });
```

Comment:
I don't think we can test this, or maybe yes

you can call `plugin.init(plugin, /*CORE LIGHTNING REQUEST*/)` you find the request on the cln docs :)

and after that check if the propriety in the plugin are in the correct place